### ISSUE TITLE
Theme doesn't display 

### ISSUE THREAD

**Issue by ronycohen** [keycloak/keycloak#34978]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

login/ui

### Describe the bug

After installing the keycloak (docker) solution and apply themes to the Realm.
![image](https://github.com/user-attachments/assets/1bfa5a87-471e-4ecd-8d65-f22b4c71457a)

The login UI page is not displaying any style : 

http://keycloak.local:8080/realms/GRC/protocol/openid-connect/auth?client_id=grc&redirect_uri=http%3A%2F%2Flocalhost%3A5173%2Flogin&state=31c6beba-b3c1-4c92-8935-47fa1ae539df&response_mode=fragment&response_type=code&scope=openid&nonce=23795dbe-1f92-43ae-82d6-5a94cbf2894f&code_challenge=JoHQc13fJMO3K3rmcqyC_qT_zz-oQlOkOZiKn5YNE8k&code_challenge_method=S256

![image](https://github.com/user-attachments/assets/7c7fdb3f-190c-402b-999e-39a7f8abfd41)

Here is the Docker Compose : 
```
services:
  keycloak:
    image: quay.io/keycloak/keycloak:latest
    environment:
      - KC_DB=postgres
      - KC_DB_URL=jdbc:postgresql://192.168.1.12:5432/keycloak  
      - KC_DB_USERNAME=keycloak
      - KC_DB_PASSWORD=keycloak
      - KC_HOSTNAME_STRICT=false
      - KC_HOSTNAME=keycloak.local
      - KC_BOOTSTRAP_ADMIN_USERNAME=admin
      - KC_BOOTSTRAP_ADMIN_PASSWORD=admin
      - KEYCLOAK_ADMIN=admin
      - KEYCLOAK_ADMIN_PASSWORD=admin
      - KC_HOSTNAME=keycloak.local
      # - KC_HTTP_RELATIVE_PATH=/
      - KC_FRONTEND_URL=http://keycloak.local:8080/
      - KC_LOG_LEVEL=debug
      - KC_HOSTNAME_STRICT_HTTPS=false
    ports:
      - "8080:8080"
    volumes:
      - ./themes:/opt/keycloak/themes
    restart: always
    command: start-dev
    #command: start
    
```

### Version

26.0.5

### Regression

- [ ] The issue is a regression

### Expected behavior

I would expect the Base or one of the custom theme I applied to the Client created.

### Actual behavior

Currently I don't have any Style applied to the Login/ui page.

### How to Reproduce?

I just installed the latest version through docker compose, created a Realm a client and a user. I did connect a Web-App to keycloak and got a login page without style.

### Anything else?

I tried to apply several custom theme : 
 
```
   volumes:
      - ./themes:/opt/keycloak/themes
```   
But even with them selected it's not working.

**Comment by ronycohen** [comment]
@thomas-mauran  how did you resolve it ?

**Comment by sschu** [comment]
Not sure, but I assume the base theme just does not contain any styling. Do the other themes work as login theme?

**Comment by ronycohen** [comment]
Nope. Other themes are not working either.

Here is more information 
![image](https://github.com/user-attachments/assets/92cecb94-8fcb-45aa-b96d-7b8870c45d85)

**Comment by thomas-mauran** [comment]
> @thomas-mauran  how did you resolve it ?

I did not, I just switched back to the default one

**Comment by sschu** [comment]
I tried to reproduce this using the provided compose file. There is indeed no styling for the `base` theme but both `keycloak` and `keycloak.v2` work for me.

**Comment by mposolda** [comment]
This looks like an incorrect use as pointed by @sschu rather than Keycloak bug . The "base" theme does not contain any CSS in it. You should use "keycloak" or "keycloak.v2" theme if you want to see any CSS styles.

I suggest to first try it without your custom themes. If it works WITHOUT your theme, but does not work WITH your theme, then it is clear that the issue is in your custom theme. Please re-open this just if you don't see any CSS even if you are NOT using your custom themes extension and still not seeing any CSS for "keycloak" or "keycloak.v2" themes.

**Comment by ronycohen** [comment]
Even with the "keycloak" or "keycloak.v2" theme I don't have any styling available....

**Comment by sschu** [comment]
@ronycohen I tried to reproduce the problem using the steps you provided and everything works. So I assume it is something in your environment and not a bug in Keycloak.

**Comment by denniskuligin** [comment]
Hey folks, I’m having a similar issue and it’s driving me a bit crazy.

I copied the base theme and renamed it to /themes/corporate/, expecting to just tweak the styles and move on. But guess what? Keycloak simply doesn’t detect it at all — no matter what I try.

parent=base in theme.properties doesn’t help.

Reloading the server, restarting the service – nothing changes.

There are no errors in the logs either 🤷‍♂️

I’m just a frontend engineer trying to make our corporate branding apply to Keycloak… but seriously, why is it so hard to get a custom theme working?

Running Keycloak 24.0.3.
Followed the documentation to the letter — still no luck.

Any idea what could be wrong or what I might be missing?

**Comment by R-N** [comment]
> I copied the base theme and renamed it to /themes/corporate/, expecting to just tweak the styles and move on. But guess what? Keycloak simply doesn’t detect it at all — no matter what I try.

@denniskuligin I'm having the same issue. Keycloak detects the theme, I've selected it in the realm configuration (GUI). However no matter what change I made to the theme file (login.ftl), no change seems to apply. I've tried letting it be empty file but the login form still renders as usual. No error, even in the logs. I'm on Keycloak 24. I've tried bitnami's and quay's, issue persists. 

theme.properties:
```
parent=base
import=common/keycloak
```

Did you find a solution?
